src/main/java/application/rest/v1/configmaps/KindActionMappingProcessor.java

```diff
@@ -0,0 +1,462 @@
+                "\n group = " + "actions.kappnav.io" + 
+                "\n namespace = kappnav and name = default");
+        }
+
+        Object kamResource = coa.listClusterCustomObject(KAM_GROUP, KAM_VERSION, KAM_PLURAL, null, 
+                             null, null, null);
+        return KAppNavEndpoint.getItemAsList(client, kamResource);
+    }
+
+    /**
+     * Get the namespace for the given KindActionMapping resource
+     * 
+     * @param kam
+     * @return the namespace for the given kam resource
+     */
+    private String getKAMNamespace(JsonElement kam) {
+        if (Logger.isEntryEnabled())
+            Logger.log(className, "getKAMNamespace", Logger.LogType.ENTRY,"");
+
+        String namespaceStr = null;
+        JsonElement metadata = (JsonObject) kam.getAsJsonObject().get(METADATA_PROPERTY_NAME);
+        if (metadata != null) {
+            JsonObject namespace= metadata.getAsJsonObject();
+            if (namespace != null) 
+                namespaceStr = namespace.get(NAMESPACE_PROPERTY_NAME).getAsString();
+        } else {
+            if (Logger.isDebugEnabled()) 
+                Logger.log(className, "getKAMNamespace", Logger.LogType.DEBUG, "kam metadata is null.");
+        }
+
+        if (Logger.isExitEnabled()) {
+            Logger.log(className, "getKAMNamespace", Logger.LogType.EXIT, "kam namespace = " + namespaceStr);
+        } 
+        return namespaceStr;
+    }
+
+    /**
+     * Check to see if the apiVersions of a component and a mapping are matching
+     * 
+     * @param apiVersion
+     * @param compApiVersion
+     * @return
+     */
+    private boolean isApiVersionMatch(String apiVersion, String compApiVersion) {
+        if (Logger.isEntryEnabled()) 
+            Logger.log(className, "isApiVersionMatch", Logger.LogType.ENTRY, "apiVerion = " + apiVersion +
+                       ", compApiVersion = " + compApiVersion);
+
+        boolean match = false;
+        if ( (apiVersion != null) && (!apiVersion.isEmpty()) &&
+             (compApiVersion != null) && (!compApiVersion.isEmpty()) ) {   
+            if (apiVersion.equals(compApiVersion)) { // group/version - exact match
+                match = true;
+            } else {
+                String grp_version[] = apiVersion.split("/");
+                String comp_grp_version[] = compApiVersion.split("/");
+
+                if ( (grp_version.length == 2) && (comp_grp_version.length == 2) ){ // group/version - wildcard match
+                    if (grp_version[0].equals(comp_grp_version[0]) || (grp_version[0].equals(WILDCARD)) )
+                        if (grp_version[1].equals(comp_grp_version[1]) || (grp_version[1].equals(WILDCARD)) ) {
+                            match = true;
+                        }
+                } else if ( (grp_version.length == 1) && (comp_grp_version.length == 1) ) { // verion only
+                    if (grp_version[0].equals(comp_grp_version[0]) || (grp_version[0].equals(WILDCARD)) ) {
```

Comment:
If you use index constants,  0 is not GROUP in this case, it's actually VERSION.  So maybe use a different constant here, such as NO_GROUP_VERSION or GROUPLESS_VERSION = 0, since this is the case for those older, built-in kinds that have version, but no group.